src/meta/proto-conv/tests/it/v029_schema.rs

```diff
@@ -0,0 +1,99 @@
+        29, 168, 6, 24, 18, 9, 146, 2, 0, 160, 6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 160, 6,
+        29, 168, 6, 24, 18, 17, 154, 2, 8, 66, 0, 160, 6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24,
+        160, 6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 32, 1, 160, 6, 29, 168, 6, 24, 10, 30, 10,
+        1, 99, 26, 17, 154, 2, 8, 34, 0, 160, 6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 32, 4,
+        160, 6, 29, 168, 6, 24, 10, 49, 10, 10, 100, 101, 99, 105, 109, 97, 108, 49, 50, 56, 26,
+        27, 218, 2, 18, 10, 10, 8, 18, 16, 3, 160, 6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 160,
+        6, 29, 168, 6, 24, 32, 5, 160, 6, 29, 168, 6, 24, 10, 49, 10, 10, 100, 101, 99, 105, 109,
+        97, 108, 50, 53, 54, 26, 27, 218, 2, 18, 18, 10, 8, 46, 16, 6, 160, 6, 29, 168, 6, 24, 160,
+        6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 32, 6, 160, 6, 29, 168, 6, 24, 10, 30, 10, 9,
+        101, 109, 112, 116, 121, 95, 109, 97, 112, 26, 9, 226, 2, 0, 160, 6, 29, 168, 6, 24, 32, 7,
+        160, 6, 29, 168, 6, 24, 10, 79, 10, 3, 109, 97, 112, 26, 64, 194, 2, 55, 202, 2, 46, 10, 3,
+        107, 101, 121, 10, 3, 118, 97, 108, 18, 9, 146, 2, 0, 160, 6, 29, 168, 6, 24, 18, 17, 154,
+        2, 8, 66, 0, 160, 6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 160,
+        6, 29, 168, 6, 24, 160, 6, 29, 168, 6, 24, 32, 8, 160, 6, 29, 168, 6, 24, 24, 9, 160, 6,
+        29, 168, 6, 24,
+    ];
+
+    let b1 = TableDataType::Tuple {
+        fields_name: vec!["b11".to_string(), "b12".to_string()],
+        fields_type: vec![TableDataType::Boolean, TableDataType::String],
+    };
+    let b = TableDataType::Tuple {
+        fields_name: vec!["b1".to_string(), "b2".to_string()],
+        fields_type: vec![b1, TableDataType::Number(NumberDataType::Int64)],
+    };
+    let fields = vec![
+        TableField::new("a", TableDataType::Number(NumberDataType::UInt64)),
+        TableField::new("b", b),
+        TableField::new("c", TableDataType::Number(NumberDataType::UInt64)),
+        TableField::new(
+            "decimal128",
+            TableDataType::Decimal(DecimalDataType::Decimal128(DecimalSize {
+                precision: 18,
+                scale: 3,
+            })),
+        ),
+        TableField::new(
+            "decimal256",
+            TableDataType::Decimal(DecimalDataType::Decimal256(DecimalSize {
+                precision: 46,
+                scale: 6,
+            })),
+        ),
+        TableField::new("empty_map", TableDataType::EmptyMap),
+        TableField::new(
+            "map",
+            TableDataType::Map(Box::new(TableDataType::Tuple {
+                fields_name: vec!["key".to_string(), "val".to_string()],
+                fields_type: vec![
+                    TableDataType::String,
+                    TableDataType::Number(NumberDataType::Int64),
+                ],
+            })),
+        ),
+    ];
+    let want = || TableSchema::new(fields.clone());
```

Comment:
The other `Tablefield`s except those that contain EmptyMap are not necessary for this test IMHO.

Reply:
ok, I will remove EmptyMap.

Reply:
Sorry, I mean, you only have to test EmptyMap. Since it is the only change in this PR...

Reply:
ok